compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       context: ./database/.docker/db
       dockerfile: Dockerfile
       args:
-        - POSTGRES_VERSION=${POSTGRES_VERSION}
+        - POSTGRES_VERSION=14.2
```

Comment:
This should stay ${POSTGRES_VERSION}, but since it's reading from your .env file, you'll need to change the POSTGRES_VERSION value from 12.5 to 14.2 in your .env.